interview-questions.md

```diff
@@ -7,60 +8,48 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
-
-  Researched answer:
-
+Your answer: Git a tool created by the same creator as Linux used for version control so multiple developers can easily code on the same project. GitHub is a website that people can create projects and repositories on which utilizes the Git technology.
 
+Researched answer: Git is software created by Linus Torvalds that can track changes in sets of files usually between developers working on the same source code. GitHub is a website, now owned by Microsoft, that hosts version control which uses Git.
 
 2. What types of evaluations will return a Boolean value?
 
-  Your answer:
-
-  Researched answer:
-
+Your answer: A Boolean value returned by checking if something is true or false. This can be used in if/else if/else statements to execute code based on specific conditions.
```

Comment:
More specifically, relational, logical, and equality operators return a Boolean value.